webpack/assets/javascripts/react_app/routes/Models/consts.js

```diff
@@ -0,0 +1,19 @@
+
+export default {
+  controller: "models",
+  autocomplete : {
+    searchQuery: null,
+    url: "/models/auto_complete_search",
+    id: "searchBar",
+    useKeyShortcuts: true
+  },
+  bookmarks: {
+    url: "/api/bookmarks",
+    canCreate: true,
+    documentationUrl: "https://theforeman.org/manuals/1.23/index.html#4.1.5Searching"
```

Comment:
Somewhere else we just use the relative part. Hopefully that will then refer to the correct version.
https://github.com/theforeman/foreman/blob/c5fbd28993fbe29f3457a329245e8975dd4c7667/webpack/assets/javascripts/react_app/constants.js#L22